cosmo_tester/test_suites/test_security/security_test_base.py

```diff
@@ -179,6 +179,10 @@ def _set_credentials_env_vars(self):
         os.environ[constants.CLOUDIFY_USERNAME_ENV] = TEST_CFY_USERNAME
         os.environ[constants.CLOUDIFY_PASSWORD_ENV] = TEST_CFY_PASSWORD
 
+    def _unset_credentials_env_vars(self):
+        del os.environ[constants.CLOUDIFY_USERNAME_ENV]
```

Comment:
it is preferable you do `os.environ.pop(constants.CLOUDIFY_USERNAME_ENV, None)` to protect against the case the env var was never updated